lms/djangoapps/django_comment_client/tests/group_id.py

```diff
@@ -95,7 +95,7 @@ def test_cohorted_topic_moderator_with_other_group_id(self, mock_request):
     def test_cohorted_topic_moderator_with_invalid_group_id(self, mock_request):
         invalid_id = self.student_cohort.id + self.moderator_cohort.id
         response = self.call_view(mock_request, "cohorted_topic", self.moderator, invalid_id)
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 500)
```

Comment:
Double checking that this should be `HttpResponseServerError` vs `HttpResponseBadRequest`?